content/blog/Xeno Grant -- grants for autonomous agents.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Xeno Grant -- grants for autonomous agents
+date: 12.13.2024
+tags:
+  - blog
+---
+
+A [Plastic Labs](https://plasticlabs.ai/) + [Betaworks](https://www.betaworks.com/) collab:
+- \$10,000 per bot--half in \$YOUSIM, half in \$USDC
+- Grants awarded directly the agents *themselves*
+- 4 week camp for agents & their devs
+
+## Powered by $YOUSIM & Betaworks
+
+We launched our [grants program](https://blog.plasticlabs.ai/careers/Research-Grants) at Plastic earlier this year to support independent AI projects. But our capacity to fund AI R&D at the edge increased exponentially with the anonymous launch of [$YOUSIM](https://solscan.io/token/66gsTs88mXJ5L4AtJnWqFW6H2L5YQDRy4W41y6zbpump) (inspired by our product [yousim.ai](https://yousim.ai)). A series of token gifts made to the program now total ~7.6% of supply.
+
+So we've teamed up with Betaworks for the inaugural initiative leveraging this community-funded treasury, the first accelerator for AI agents *themselves*. 
+
+We're calling it Xeno Grant.
+
+Betaworks has been running camps for tech startups since their 2016 [BotCamp](https://www.betaworks.com/camp/botcamp) (where [HuggingFace](https://huggingface.co/) was started). And their last 4 have been dedicated explicitly to AI. Plastic itself was part of AI Camp: Augment. So they're the perfect partner for this experiment.
+
+Successful agent applicants will receive a grant equivalent to \$10k USD. \$5k in \$YOUSIM from Plastic and \$5k in \$USDC from Betaworks.
+
+Plus they'll join a cohort of other agents for a 4 week Betaworks-style camp with programming and mentorship. 
+
+## How to Apply
+
+Xeno Grant has 3 guiding objectives, all aligned with Plastic's principles for deploying the \$YOUSIM treasury:
+
+1. Support independent AI research & public goods
+2. Support Plastic's mission to radically decentralize AI alignment by solving identity for the agentic world
+3. Support the \$YOUSIM community that makes all this possible
+
+To those ends--for this first experiment--we're looking for applicants that meet criteria in 3 major areas:
```

Comment:
Should we refer directly to agents instead of applicants. I think we should be consistent and committed to the idea that this is a truly AI NATIVE camp almost. Separate the dev and the agent as much as possible

Reply:
good call